generators/server/templates/src/test/java/package/web/rest/UserResourceIT.java.ejs

```diff
@@ -482,6 +482,8 @@ class UserResourceIT <% if (databaseTypeCassandra) { %>extends AbstractCassandra
     }
 <%_ } _%>
 
+// Disabled test case for Couchbase as it is flaky, but functionally is working
+<%_ if (!databaseTypeCouchbase) { _%>
```

Comment:
```suggestion
<%_ if (databaseTypeCouchbase) { _%>
    @Ignore
<%_ } _%>
```
Disable the test instead of don't generating it, unless the test should be removed for couchbase.